docs/cado-response/deploy/gcp/gcp-deploy.md

```diff
@@ -53,7 +53,9 @@ If you have not received the above items, reach out to sales@cadosecurity.com fo
     :::
 10. Deploy by running the following commands in the directory `gcp/`
     `terraform init` followed by `terraform plan`.  Once you confirm the plan looks correct, you can then run `terraform apply -var-file=gcpVars.tfvars`
-11. Find your Cado instance in the Compute Engine portal, navigate to its public IP address in the browser
-12. Login with:
+11. Enable GCP Services **Compute Engine**, **Service Accounts** and **Cloud Build** in the [Cloud Build settings page](https://console.cloud.google.com/cloud-build/settings/service-account?)
+<img src="/img/cloudbuild-settings.png" alt="Cloud Build Settings" width="500"/>
+12. Find your Cado instance in the Compute Engine portal, navigate to its public IP address in the browser
```

Comment:
We could embed the link here: https://console.cloud.google.com/compute/instances